caluma/caluma_workflow/tests/test_case.py

```diff
@@ -674,7 +674,7 @@ def test_cancel_suspend_resume_case(
     expected_work_item_status,
     error,
 ):
-    if use_api:
+    if not use_api:
```

Comment:
`use_api` means use the graphql api, maybe we should reconsider the naming of that param..

Reply:
True, will do!